README.md

```diff
@@ -18,90 +28,66 @@ One of the most important aspects of collaboration is knowing who else you're wo
 
 ### Where is each user within the application?
```

Comment:
Can we take these headings down one or two levels? It'll look less text-heavy I think.